package.json

```diff
@@ -9,7 +9,7 @@
   "//": "(Blame C++)",
   "scripts": {
     "preinstall": "node scripts/configure.js",
-    "install": "./gyp_llnode && make -C out/",
+    "install": "./gyp_llnode && gmake -C out/ || make -C out/",
```

Comment:
Logic bug: this is parsed as `(./gyp_llnode && gmake -C out/) || make -C out/` which makes `make -C out/` run even when gyp_llnode fails.  I speculate the pile up of errors is going to look confusing to users so I'd suggest adding parentheses.